apps/files/tests/test_models.py

```diff
@@ -68,6 +68,18 @@ def test_get_absolute_url(self):
                     'delicious_bookmarks-2.1.072-fx.xpi?src=src')
         assert url.endswith(expected), url
 
+    def test_get_url_path(self):
+        file_ = File.objects.get(id=67442)
+        assert file_.get_url_path('src') == \
```

Comment:
This really makes me wonder why `get_url_path` exists. It is named incorrectly and has the same value as the conventional method.